src/geb/geb.lisp

```diff
@@ -130,7 +181,7 @@ u
                morph dom)
         (comp morph (commutes (mcadr dom) (mcar dom))))))
 
-(-> !-> (substobj substobj) substobj)
```

Comment:
uhhhhhh, this feels wrong, given it literally matches `a` if anything we may need to generalize `b` to `catsubstobj`.